### PR TITLE
Header Markup

### PR DESCRIPTION
**Pull request by taliahm**
Added the HTML for the header as per designs. 💃🏾

Had some trouble with the <nav> would love your thoughts on that.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,17 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <header></header>
+    <header>
+        <nav>
+            <ul>
```
Comment (darshanasen) [inline index.html]: Update ul to ol (for better semantic structure)

File: index.html [hunk 2]
```diff
@@ -7,7 +7,17 @@
...
+                <li>Thing One</li>
+                <li>Thing Two</li>
+                <li>Thing Three</li>
+                <li>Thing Four</li>
+            </ul>
+        </nav>
+        <h1>STUFF N THINGS!!!!</h1>
```
Comment (darshanasen) [inline index.html:19]: Update "N" to "&"

**Review by darshanasen — CHANGES_REQUESTED** [review]
GAWGEOUS code. I have some comments about a few HTML element/content updates. 🌻

File: index.html (hunk 2 above)
Comment (taliahm) [reply index.html:19]: As per our discussion, we are not making this change!

Comment (darshanasen) [reply index.html:19]: Fine.

**Review by darshanasen — APPROVED** [review]
Excellent changes, my friend. Thank you!

**Pull request merged**